cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/common/tools/ResourcesPluginUtil.java

```diff
@@ -83,6 +96,10 @@ public static void refreshConfigurationProject() {
    */
   public static IProject getGeneratorConfigurationProject() throws GeneratorProjectNotExistentException, CoreException {
 
+    if (Files.exists(getTemplateSetDirectory().toPath())) {
+      generatorProjExists = false;
+      return null;
+    }
```

Comment:
This is done in line number 134. The thing is, after the upgrade the template-set will always be used for generation as long as the template-set folder will not be removed. Even if a new templates project is imported.  This is done because the template-set structure should be preferred. Also, we do not yet support upgrading multiple templates projects. Is this a problem? Maybe this issue should be solved in a new ticket? @maybeec